src/ReplGPT.jl

```diff
@@ -4,13 +4,62 @@ import OpenAI
 import ReplMaker
 import Markdown
 
+@static if VERSION >= v"1.6"
+    using Preferences
+end
```

Comment:
You have also set the minimum Julia version to 1.7 in `Project.toml` so this conditional is not necessary.

By the way, setting `julia = "1.6"` in `Project.toml` allows Julia 1.6, 1.7, etc. Anything above 1.6. Setting 1.6.0 would allow only 1.6.1, 1.6.2, etc

Reply:
Cool, thanks! I've extended the supported range to 1.6 and greater (good to support the LTS) and removed the `VERSION` checks